src/pca.jl

```diff
@@ -54,19 +54,9 @@ Base.propertynames(pca::PCA, private::Bool = false) = (
 #    :rotation,
 )
 
-Base.show(pca::PCA;
-        ndigitsmat=2, ndigitsvec=2, ndigitscum=4,
-        covcor=true, loadings=true, variances=false, stddevs=false) =
-        Base.show(Base.stdout, pca,
-                    ndigitsmat=ndigitsmat,
-                    ndigitsvec=ndigitsvec,
-                    ndigitscum=ndigitscum,
-                    covcor=covcor,
-                    loadings=loadings,
-                    variances=variances,
-                    stddevs=stddevs)
-
-function Base.show(io::IO, pca::PCA;
+Base.show(io::IO, pca::PCA; kwargs...) = Base.show(io, MIME"text/plain"(), pca; kwargs...)
+
```

Comment:
Why is the second argument `MIME"text/plain"()` here and `MIME"text/plain"` in other places?  I see that both cases are mentioned in the documentation for `Base.show` but it is not clear to me what the distinction is.

Reply:
Good question! `MIME"text/plain" ` is a type, so the signature includes that. But we only use it for multiple dispatch, hence no binding. But when calling the function, we need to instantiate that type.

Reply:
I suppose we could also dispatch on the type itself with `::Type{MIME"text/plain"}` but that gives me a headache.

Reply:
The answer makes sense - it was the sort of thing I was anticipating.  No need to change anything.